capstone/src/main/java/com/google/univiz/api/SearchResourceImpl.java

```diff
@@ -0,0 +1,31 @@
+package com.google.univiz.api;
+
+import com.google.common.collect.Lists;
+import com.google.univiz.api.representation.CollegeId;
+import com.google.univiz.api.representation.SearchData;
+import com.google.univiz.api.representation.SuggestionResponse;
+import com.google.univiz.api.resource.SearchResource;
+import java.util.List;
+import java.util.stream.Collectors;
+import javax.inject.Inject;
+
+final class SearchResourceImpl implements SearchResource {
+  private final SuggestionDataApi suggestionApi;
+
+  @Inject
+  public SearchResourceImpl(SuggestionDataApi suggestionApi) {
+    this.suggestionApi = suggestionApi;
+  }
+
+  @Override
+  public List<SearchData> getSearchSuggestions(String partialCollegeName) {
+    if (partialCollegeName == null || !partialCollegeName.matches("[a-zA-Z]+")) {
+      return Lists.newArrayList(SearchData.create("", CollegeId.create(0)));
+    }
+    System.out.println(partialCollegeName);
```

Comment:
Let's not leave commit sysouts to the repo. If you want to log something, please use https://github.com/google/flogger

Reply:
removed!